inpage/src/EthereumApi.ts

```diff
@@ -0,0 +1,101 @@
+import assert from './helpers/assert';
+import randomId from './helpers/randomId';
+
+export default class EthereumApi {
+  #networkUrl = 'http://127.0.0.1:8545';
+  #testAddress = '0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266';
```

Comment:
What do you think about how we will initialize this address when it's not hardcoded?  

For example, for an EOA do you think the user would be able to enter their pk in the popup?

Reply:
I think there should be a normal onboarding process, which a dapp could replace by directly assigning a private key / passphrase.